app/components/ui/Banner/Banner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './Banner.scss';
+
+export default function Banner() {
+  return (
+    <div className={styles.bannerContainer}>
+      <strong>CORONAVIRUS (COVID-19) UPDATE: </strong>
+      <span>
```

Comment:
Why does this `span` need to exist? I feel like this shouldn't be necessary.

Reply:
oops ya you're right! I'll go ahead and merge this in case we want to get this temporary one out tonight and will clean it up once I get actual designs from derek tomorrow.